### ISSUE TITLE
Tool for resizing FAT partitions

### ISSUE THREAD

**Issue by pali** [dosfstools/dosfstools#91]
dosfstools is missing some tool which can resize existing FAT partition. Similar to ext*'s resize2fs.

**Comment by pali** [comment]
Also there is missing a tool which can change/extend maximal number of entries in the FAT12/16 root directory.

**Comment by fstirlitz** [comment]
https://sourceforge.net/projects/fatresize/ already exists.

**Comment by pali** [comment]
Yes, but it is not part of dosfstools, depends on libparted, does not support FAT12 and does not support changing number of entries in root directory.

**Comment by stikonas** [comment]
> Yes, but it is not part of dosfstools, depends on libparted, does not support FAT12 and does not support changing number of entries in root directory.

Also, since it depends on libparted, it does not work on setups unsupported by libparted, e.g. LVM logical volume, RAID, etc...
